src/config.ts

```diff
@@ -272,3 +344,20 @@ async function globFiles(pattern: string): Promise<string[]> {
   });
   return await globber.glob();
 }
+
+function sort_object(o: any): any {
```

Comment:
hm, tbh I wouldn’t go this far, but it doesn’t hurt either.

Reply:
I just want to make sure everything is normalised to a fix order, so if the users only change the order it should not save a new cache.